ci-operator/config/openshift/release/openshift-release-master__nightly-4.7.yaml

```diff
@@ -321,6 +329,71 @@ tests:
         requests:
           cpu: 100m
     workflow: openshift-e2e-powervs-upi
+- as: e2e-cnv-deploy
+  interval: 12h
+  steps:
+    cluster_profile: azure4
+    test:
+    - as: test
+      cli: latest
+      commands: make deploy_test
+      credentials:
+      - mount_path: /var/run/cnv-ci-brew-pull-secret
+        name: cnv-ci-brew-pull-secret
+        namespace: test-credentials
+      env:
+      - default: '|7820aea2-0d75-11e7-9259-28d244ea5a6d.hhav.f63e13'
```

Comment:
Should this not be a secret with the brew configuration?

Reply:
This is just the registry username.
The secret token itself is stored at `/var/run/cnv-ci-brew-pull-secret/token`
Should the username also be a secret? It is configured as an env var also for the job for our repo here:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-release-4.9.yaml#L58-L59